app/javascript/src/components/ItemList/ItemListTags.jsx

```diff
@@ -10,7 +10,7 @@ export const ItemListTags = ({ availableTags, selectedTags, onClick }) => {
       data-no-instant
       onClick={e => onClick(e, tag)}
     >
-      #{tag}
+      {`#${tag}`}
```

Comment:
Ok, I'm curious about this? Was it Prettier or you did that on purpose? :)

Reply:
I did it on purpose to fix an eslint error.

Reply:
Really? It's just weird to me but that's alright. Thanks!

Reply:
Now I get why we have to use this approach. Dealing with whitespaces around words is a nightmare :D Shouldn't be a problem but for me it causes headache in Jest snapshot tests.